docs/home/api-info/webhooks.mdx

```diff
@@ -20,37 +20,39 @@ To enable this integration approach with Metriport:
 1. Set this endpoint URL on the [Developers page](https://dash.metriport.com/developers) in the developer dashboard,
    or via the [Update Settings endpoint](/home/api-reference/settings/post-settings);
 1. This will generate a webhook key that should be used to authenticate requests on
-   your app's endpoint (webhook).
+   your app's endpoint (webhook) - see [authentication](#authentication) and 
+   [generating a new webhook key](#generating-a-new-webhook-key) below.
 
 General requirements for the endpoint:
 
 1. Public endpoint accessible from the internet;
 1. Does not do an HTTP redirect (redirects will not be followed);
 1. Accepts a `POST` HTTP request;
-1. Verifies requests by comparing the HTTP header `x-webhook-key` with the webhook key;
+1. Verifies requests using the HTTP header `x-metriport-signature` - see [authentication](#authentication) below;
 1. Accepts and responds to a [`ping` message](#the-ping-message);
+1. Be [idempotent](https://en.wikipedia.org/wiki/Idempotence) - it should accept being called more
```

Comment:
Some thoughts/Qs:

- Is the "idempotent" requirement essentially just telling the cx that we always expect a `200` HTTP response? Perhaps we should just tell them that, since we don't really care what they do on their end?
- We've had problems where cx webhook endpoints take a long time to process the payload, and we timeout waiting for the response. Should we instruct them to process the payload async and return us `200` right away?

Reply:
> Is the "idempotent" requirement essentially just telling the cx that we always expect a 200 HTTP response? Perhaps we should just tell them that, since we don't really care what they do on their end?

No, saying it should be idempotent is not the same thing as saying it should always return 200. They need to design their endpoint in a way that allows it being called multiple times w/ the same payload w/o breaking their app.

> We've had problems where cx webhook endpoints take a long time to process the payload, and we timeout waiting for the response. Should we instruct them to process the payload async and return us 200 right away?

Good point, added!